### DISCUSSION TITLE
Problem with server component and client component

### DISCUSSION THREAD

**Discussion by ask4fusora** [vercel/next.js#66824] [Help]
### Summary

So I started my app with this prompt:
`bunx create-next-app@latest nextjs --typescript --tailwind --eslint --use-bun --src-dir --app --import-alias "@/*"`
Which is pretty much the default.
Next, `bun run dev`, and it showed me the default landing page as expected.

The problem start here, when I add `"use client"` to the top of `src/app/page.tsx`, on the browser, the content of the page appear for a short while and then everything inside of the `<body>` tag get deleted [short vid](https://github.com/vercel/next.js/assets/75573917/304b7c8c-71bc-496e-af7c-17706c408299).

So I deleted `"use client"`, create a new file `src/app/Button.tsx`, and import it to `src/app/page.tsx`.
The 2 files' content is of the following:

`src/app/Button.tsx`
```tsx
"use client"

const Button: React.FC = () => <button>Click me</button>

export default Button
```

`src/app/page.tsx`
```tsx
import Button from "./Button";

export default function Home() {
 return <Button /> 
}
```

It showed me this error after the desired output was cleared after a short while, [short video](https://github.com/vercel/next.js/assets/75573917/bba338e7-0381-44a5-81fb-9dd99fd6e985).

Can you help me how to fix this please.

### Additional information

```js
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 11 Home Single Language
  Available memory (MB): 15790
  Available CPU cores: 16
Binaries:
  bun: 1.0.13
  Node: 22.3.0
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.2.4 // Latest available version is detected (14.2.4).
  eslint-config-next: 14.2.4
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.4.5
Next.js Config:
  output: N/A
```

**Comment by icyJoseph** [comment]
Try removing `.next`, or restarting the server.

**Reply by ask4fusora** [reply]
I had tried both, it did not work.
I noticed that there's some kind of delay before the problems occured, so I tried the build step.
The production build worked fine.

So I think it has something to do with react's double render in dev mode.
